### PR TITLE
stand-alone (electron) app

### PR DESCRIPTION
**Pull request by yoh1496**
To make app-uc-unit-manager can be used freely from web server,
I implemented some codes to launch app-uc-unit-manager in electron.

How about this?

### CODE REVIEW

**Review by dixonsiu — APPROVED** [review]
Great job. Thank you.

File: main.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const { app, BrowserWindow } = require("electron");
+
+let mainWindow = null;
+
+app.on("ready", () => {
+  mainWindow = new BrowserWindow();
+  mainWindow.loadFile(__dirname + "/src/html/login.html");
```
Comment (dixonsiu) [inline main.js:7]: Please put comment or add information in README that the path must be modified for Personium Cell or Electron.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "electron main.js",
+    "build:win": "electron-builder"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/personium/app-uc-unit-manager.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (tochi-y) [inline package.json]:
```suggestion
  "license": "Apache License 2.0",
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "app-uc-unit-manager",
+  "version": "1.0.0",
```
Comment (tochi-y) [inline package.json:3]: Release version needs to be considered.

**Review by tochi-y — CHANGES_REQUESTED** [review]
Thank you for your great work! Please check the comments.

**Comment by yoh1496** [comment]
@tochi-y 
I tested that `login.js` works if it is launched in Web Browser.
Could you merge this?

**Comment by tochi-y** [comment]
@yoh1496 After merging #178, conflicts have occurred. Can you fix this?

**Comment by yoh1496** [comment]
I rebased and solved conflicts

File: package.json (hunk 2 above)
Comment (tochi-y) [inline package.json]:
```suggestion
  "license": "Apache License 2.0",
```

**Review by tochi-y — APPROVED** [review]

**Pull request merged**